external/anomaly/tools/sample.py

```diff
@@ -0,0 +1,278 @@
+import argparse
+import os
+import logging
+import importlib
+import time
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+from pathlib import Path
+from typing import List, Union
+
+from ote_anomalib.data import LabelNames
+
+from ote_sdk.entities.annotation import (
+    Annotation,
+    AnnotationSceneEntity,
+    AnnotationSceneKind,
+)
+from ote_sdk.entities.dataset_item import DatasetItemEntity
+from ote_sdk.entities.datasets import DatasetEntity
+from ote_sdk.entities.image import Image
+from ote_sdk.entities.label import Domain, LabelEntity
+from ote_sdk.entities.scored_label import ScoredLabel
+from ote_sdk.entities.shapes.rectangle import Rectangle
+from ote_sdk.configuration.helper import create
+from ote_sdk.entities.inference_parameters import InferenceParameters
+from ote_sdk.entities.label_schema import LabelSchemaEntity
+from ote_sdk.entities.model import ModelEntity, ModelOptimizationType, ModelPrecision, ModelStatus, OptimizationMethod
+from ote_sdk.entities.model_template import TargetDevice, parse_model_template
+from ote_sdk.entities.optimization_parameters import OptimizationParameters
+from ote_sdk.entities.resultset import ResultSetEntity
+from ote_sdk.entities.subset import Subset
+from ote_sdk.entities.task_environment import TaskEnvironment
+from ote_sdk.entities.train_parameters import TrainParameters
+from ote_sdk.usecases.tasks.interfaces.export_interface import ExportType
+from ote_sdk.usecases.tasks.interfaces.optimization_interface import OptimizationType
+
+from pandas.core.frame import DataFrame
+from tqdm import tqdm
+
+from anomalib.datasets.anomaly_dataset import make_dataset
+
+
+class OTEAnomalyDatasetGenerator:
```

Comment:
How different is this compared to `tests/helpers/dataset.py`? Would be better to modify that and import it via `from tests.helpers.dataset import OTEAnomalyDatasetGenerator`. Otherwise, it would be hard to maintain the two versions.

Reply:
It's bad practic to import from tests, tests directory not added to pacakge and need to set PYTHONPATH for this import.
I moved `tests/helpers/dataset.py`  to `ote_anomalib/data/helpers.py`  and removed `OTEAnomalyDatasetGenerator` from sample.py.

Reply:
yeah, fair point.